webpack/scenes/ContentViews/ContentViewsActions.js

```diff
@@ -8,11 +8,15 @@ import CONTENT_VIEWS_KEY, {
 import { getResponseErrorMsgs } from '../../utils/helpers';
 import { renderTaskStartedToast } from '../Tasks/helpers';
 
-export const createContentViewsParams = extraParams => ({
-  organization_id: orgId(),
-  nondefault: true,
-  ...extraParams,
-});
+export const createContentViewsParams = (extraParams) => {
```

Comment:
export const createContentViewsParams = (extraParams) => ({
    organization_id: orgId(),
    nondefault: !extraParams?.include_default,
    ...extraParams,
});

Reply:
So the API doesn't work the way we want it to with nondefault:false. That's why we remove the key from params instead of setting it to false.